src/Controller/RegionController.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Region;
+use Doctrine\ORM\EntityManagerInterface;
+use phpDocumentor\Reflection\Types\Integer;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Bundle\SecurityBundle\Security;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use function PHPUnit\Framework\isEmpty;
+
+class RegionController extends AbstractController
+{
+    #[Route('/region', name: 'korv_region_create')]
+    public function index(Request $request, EntityManagerInterface $entityManager): Response
```

Comment:
Index to create a new region ? Shouldn't it be a function new ? or function create ?